lib/manager/bazel/extract.ts

```diff
@@ -244,6 +244,7 @@ export function extractPackageFile(content: string): PackageFile | null {
       dep.versioning = dockerVersioning.id;
       dep.datasource = datasourceDocker.id;
       dep.lookupName = repository;
+      dep.registryUrls = [registry];
```

Comment:
Is it a mandatory field?

Reply:
Wondering if it should be `if(registry) dep.registryUrls = registry`

Reply:
yes, it's mandatory, see condition above.